src/jit/flowgraph.cpp

```diff
@@ -23229,11 +23229,7 @@ GenTreePtr Compiler::fgInlinePrependStatements(InlineInfo* inlineInfo)
                         //
                         // Chase through any GT_RET_EXPRs to find the actual argument
                         // expression.
-                        GenTree* actualArgNode = argNode;
-                        while (actualArgNode->gtOper == GT_RET_EXPR)
-                        {
-                            actualArgNode = actualArgNode->gtRetExpr.gtInlineCandidate;
-                        }
+                        GenTree* actualArgNode = argNode->gtRetExprVal();
```

Comment:
You may want to call this getRetExprVal() as typically gtXXX() are simple accessors, and we even have  the field named gtRetExpr.

Reply:
But is up to you, I see that we have other methods that do similar things and start with gt